src/backend/metal/src/map.rs

```diff
@@ -237,16 +237,13 @@ pub fn map_format(format: Format, is_target: bool) -> Option<MTLPixelFormat> {
             Int   => RGBA8Sint,
             Uint  => RGBA8Uint,
             Inorm => RGBA8Snorm,
-            Unorm => if is_target {
-                         BGRA8Unorm
```

Comment:
there surely were reasons for this, even if it's definitely cleaner in the PR.
@fkaa do you recall why BGRA was used here?